tests/attestation_cert_apis/host/host.cpp

```diff
@@ -141,5 +142,12 @@ int main(int argc, const char* argv[])
     result = oe_terminate_enclave(enclave);
     OE_TEST(result == OE_OK);
     OE_TRACE_INFO("=== passed all tests (tls)\n");
+#else
+    // this test should not run on any platforms where OE_USE_LIBSGX is not
+    // defined
+    OE_UNUSED(argc);
+    OE_UNUSED(argv);
+    OE_TRACE_INFO("=== tests skipped when built with OE_USE_LIBSGX=OFF\n");
+#endif
```

Comment:
you can return `SKIP_RETURN_CODE` for the non-SGX chase to indicate that it's skipped.

Reply:
good idea. Added